libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1111,4 +1118,18 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         }
     }
+
+    protected void setAttributeValuesIfNotDefault(AztecAttributes attributes, MediaFile mediaFile) {
+        if (!mediaFile.getTitle().equalsIgnoreCase(DEFAULT_MEDIA_TITLE)) {
```

Comment:
I never received `null` from `mediaFile.getTitle()` when testing images without a title set, but `new MediaFile().getTitle()` does return `null`.  I added a null-safe check with 4507ea880326ba1eedd49a737c894fcaa9685b42 just to be sure.